src/video.cpp

```diff
@@ -232,6 +232,7 @@ void CVideo::init_window()
 
 	// Add any more default flags here
 	window_flags |= SDL_WINDOW_RESIZABLE;
+    window_flags |= SDL_WINDOW_ALLOW_HIGHDPI;
```

Comment:
This line should be indented with tabs.

Reply:
F**k... Again? 😃

Reply:
Fixed - 00210065632ee7397699e7e9abf36b9822cbd69e